backend/app/environment_platform_service_configurations/schemas/glue_schema.py

```diff
@@ -0,0 +1,7 @@
+from .config_schema import BaseEnvironmentPlatformServiceConfigurationDetail
+
+
+class AWSGlueConfig(BaseEnvironmentPlatformServiceConfigurationDetail):
+    database_name: str
+    bucket_name: str
```

Comment:
Clarify that it refers to a logical bucket identifier
```suggestion
    bucket_identifier: str
```